FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -111,8 +171,12 @@ class AzParseState(Enum):
             ,"sortedTitle":"(?P<sorted_title>[^"]+)"                            # Sorted Title
         """, re.VERBOSE)
 
+    film_id_by_title = {}
+
     def __init__(self, festival_data):
         HtmlPageParser.__init__(self, festival_data, debug_recorder, 'AZ', debugging=False)
+        self.config = Config().config
+        self.max_short_duration = datetime.timedelta(minutes=self.config['Constants']['MaxShortMinutes'])
```

Comment:
Please use the config in the file keeper class too, viz. `FestivalRootDirectory`.